src/xfem/4C_xfem_xfluid_contact_communicator.cpp

```diff
@@ -1478,7 +1478,7 @@ void XFEM::XFluidContactComm::create_new_gmsh_files()
 #endif
 
   std::vector<int> g_sum_gps(5);
-  fluiddis_->get_comm().SumAll(sum_gps_.data(), g_sum_gps.data(), 5);
+  Core::Communication::sum_all(sum_gps_.data(), g_sum_gps.data(), 5, fluiddis_->get_comm());
```

Comment:
There is no resize operation on `sum_gps_`, so its size is not going to be 5. This invokes UB.

Reply:
I now resized this vector appropriately. Maybe this fixes one of the famous random xfluid failures.